build/dockerfiles/rhel.install.sh

```diff
@@ -8,7 +8,36 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all && \
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
+       microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all; \
+elif [ $architecture == "s390x" ]; then \
+       microdnf install -y git findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
+       microdnf install -y gcc gcc-c++ make automake; \
+       wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz; \
+       tar xvzf jq-1.5.tar.gz; \
+       cd jq-1.5 || exit; \
+       ./configure && make && make install; \
+       cd "$HOME" || exit; \
+       export GOVERSION=1.13; \
```

Comment:
GO 1.13 isn't supported yet. https://access.redhat.com/containers/#/registry.access.redhat.com/rhel8/go-toolset 
Please use GO 1.12 as that's the latest supported version in RHCC.